tests/unit/models/package-info-cache-test.js

```diff
@@ -80,7 +80,7 @@ describe('models/package-info-cache.js', function() {
 
     beforeEach(function() {
       projectPath = path.resolve(addonFixturePath, 'simple');
-      packageJsonPath = path.join(projectPath, 'package.json');
+      packageJsonPath = path.normalize(`${projectPath}/package.json`);
```

Comment:
Third repetition of the same comment - I *really* believe that we should be using path.join() as a best practice.

Reply:
this one I am fine with changing.